api/docs/http_report.md

```diff
@@ -44,7 +44,7 @@ Here is an example of a PUT request. (Note that the content-length is not correc
 example is formatted for readability)
 
     PUT /puppet/v3/report/kermit.com?environment=production HTTP/1.0
-    ContentType: text/pson
+    Content-Type: application/json
```

Comment:
The content section refers a couple times to it being a `pson-serialization` of Puppet::Transaction::Report` - suggest updating to `json-serialization`